dune-workspace

```diff
@@ -0,0 +1,6 @@
+(lang dune 1.1)
+
+; Add custom flags here. Default developer profile is `dev`
+(env
+ (dev     (flags :standard -rectypes -w -9-27-50))
+ (release (flags :standard -rectypes)))
```

Comment:
I don't get it. We are silencing some warnings in dev mode but not in release mode? Why would we want users to see warnings that we don't see?

Reply:
The `env` profile uses a much stricter set of warnings than the `release` one.

Reply:
I don't want to look up the dune defaults to figure out which warnings are on, can we do like in configure and use `+a...`?

Reply:
We could do our own profile indeed, but here I think I do prefer relying on Dune's. It is one thing less to worry about, and profiles can be always be printed easily.